tests/test_docstring.py

```diff
@@ -83,3 +83,33 @@ class IncludesDocstring:
 
     assert "Just a title" in result
     assert "description" in result
+
+
+@pytest.mark.help
+@backends
+def test_docstring_being_used_but_not_parsed(backend, capsys, monkeypatch):
+    # Pretend docstring-parser is not installed.
+    # cappa.base.command shadows the cappa.command module,
+    # so we have to import it manually.
+    import importlib
+    cappa_command = importlib.import_module("cappa.command")
+    monkeypatch.setattr(cappa_command, "docstring_parser", None)
+
+    @cappa.command()
+    @dataclass
+    class UnparsedDocstring:
+        """Summary.
+        
+        Example:
+
+        Body.
+        """  # noqa: D412
```

Comment:
I'm not sure the noqa comment is necessary. I had a warning in my editor but it's possible that it's coming from my user configuration.